### PR TITLE
Upgradle to 6.7.1

### PR DESCRIPTION
**Pull request by lptr**

### CODE REVIEW

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -11,17 +11,17 @@
       </trusted-artifacts>
       <trusted-keys>
          <trusted-key id="0785b3eff60b1b1bea94e0bb7c25280eae63ebe5" group="^org[.]apache[.]httpcomponents($|([.].*))" regex="true"/>
+         <trusted-key id="2db4f1ef0fa761ecc4ea935c86fdc7e2a11262cb" group="commons-codec"/>
```
Comment (wolfs) [inline gradle/verification-metadata.xml:14]: What is the reason for moving those entries around?

File: gradle/verification-metadata.xml [hunk 2]
```diff
@@ -35,6 +35,31 @@
             <pgp value="7615ad56144df2376f49d98b1669c4bb543e0445"/>
          </artifact>
       </component>
+      <component group="com.google.guava" name="failureaccess" version="1.0.1">
```
Comment (wolfs) [inline gradle/verification-metadata.xml:38]: Those entries shouldn't be necessary and are covered by the accepted keys above. Is that not the case? I think adding only the conventions plugin should be enough.

**Review by wolfs — COMMENTED** [review]
The upgrade looks good. I think we need much fewer changes in the verification metadata file.

File: gradle/verification-metadata.xml (hunk 1 above)
Comment (lptr) [reply gradle/verification-metadata.xml:14]: I'm assuming that they are now ordered in some way.

Comment (lptr) [reply gradle/verification-metadata.xml:14]: Yeah, it looks like they are now ordered alphabetically by `id`.

File: gradle/verification-metadata.xml (hunk 2 above)
Comment (lptr) [reply gradle/verification-metadata.xml:38]: OK. I wonder why the automated task doesn't just skip them then...

File: gradle/verification-metadata.xml (hunk 1 above)
Comment (lptr) [reply gradle/verification-metadata.xml:14]: TESTING

File: gradle/verification-metadata.xml [hunk 3]
```diff
@@ -50,9 +55,9 @@
             <pgp value="19beab2d799c020f17c69126b16698a4adf4d638"/>
          </artifact>
       </component>
-      <component group="org.gradle" name="test-retry-gradle-plugin" version="1.1.1">
-         <artifact name="test-retry-gradle-plugin-1.1.1.jar">
-            <sha256 value="3d5fc32376b92bd87f405ca649b8bab4333afc1e03954945f26ab657e0c1fedf" origin="Generated by Gradle because artifact wasn't signed"/>
+      <component group="org.gradle" name="test-retry-gradle-plugin" version="1.1.8">
```
Comment (wolfs) [inline gradle/verification-metadata.xml:58]: The test retry plugin entry isn't necessary any more, since it is now signed. Please remove it.

Comment (lptr) [reply gradle/verification-metadata.xml]: Removed.

**Review by wolfs — APPROVED** [review]
LGTM!

**Pull request merged**